Sources/SwiftProtobuf/UnknownStorage.swift

```diff
@@ -16,7 +16,12 @@
 
 import Foundation
 
+/// Contains any unknown fields in a decoded message (that is, fields that were
+/// sent on the wire but were not recognized by the generated message
+/// implementation).
```

Comment:
Do you want to mention it also has fields where the data type was wrong for the message's definition?